tests/unit/test_apm_config/test_apm_config.py

```diff
@@ -539,104 +534,6 @@ def test_calculate_certificates_ao_prod_trustedpath_file_present(self, mocker):
         mock_get_public_cert.configure_mock(return_value="foo")
         assert apm_config.SolarWindsApmConfig()._calculate_certificates() == "bar"
 
-    def test_calculate_logs_enabled_no_collector_enabled(self, mocker):
```

Comment:
Removing this test coverage because calculating metrics/logs "enabled" no longer depends on collector.